### PR TITLE
fix: remove replay protection disabling requirement

### PR DESCRIPTION
**Pull request by pablocarle**
Describe your pull request here:

Update requirements for passtickets

List the file(s) included in this PR:

### CODE REVIEW

**Review by balhar-jakub — APPROVED** [review]

**Review by achmelo — APPROVED** [review]

**Comment by dkelosky** [comment]
A couple of discussion points:
* should we add a note in the doc that states how we used to require replay protection be disabled? 
* for folks that previously had added `APPLDATA('NO REPLAY PROTECTION')` - should they revert this change?

For the first bullet, I think the answer is "no", but I'm curious for other's input.  For the second, I'm not sure.

File: docs/user-guide/api-mediation/configuration-extender-passtickets.md [hunk 1]
```diff
@@ -172,52 +178,55 @@ You configured Zowe to use PassTickets using ACF2.
...
+```tss
 TSS ADDTO(department) PTKTDATA(IRRPTAUT) 
 ```
-3. Define PassTicket for application ID _applid_ without replay protection.
 
-```
+3. Define PassTicket for application ID _applid_
+
+```tss
 TSS ADDTO(NDT) PSTKAPPL(<applid>) SESSKEY(<key-description>) SIGNMULTI
```
Comment (Shobhajayanna) [inline docs/user-guide/api-mediation/configuration-extender-passtickets.md:206]: SIGNMULTI should be removed, SIGNMULTI is for 'no reply protection'

**Comment by dkelosky** [comment]
Fix #3711

File: docs/user-guide/api-mediation/configuration-extender-passtickets.md [hunk 2]
```diff
@@ -61,106 +62,111 @@ To validate if a PassTicket is already defined, use the commands that correspond
...
 <summary> Click here for details about configuring Zowe to use PassTickets using ACF2. </summary>
 
+1. In your ESM command line interface or other security environment, define the application session key by entering the following commands, if the session key is not already defined.
 
-1.	In your ESM command line interface or other security environment, define the application session key by entering the following commands, if the session key is not already defined. 
-
-```
+```acf2
 SET PROFILE(PTKTDATA) DIV(SSIGNON)
 INSERT <applid> SSKEY(<key-description>) MULT-USE
```
Comment (Shobhajayanna) [inline docs/user-guide/api-mediation/configuration-extender-passtickets.md:138]: MULT-USE is for no reply protection, it should be removed

Comment (pablocarle) [reply docs/user-guide/api-mediation/configuration-extender-passtickets.md]: Thanks, updated

File: docs/user-guide/api-mediation/configuration-extender-passtickets.md (hunk 1 above)
Comment (pablocarle) [reply docs/user-guide/api-mediation/configuration-extender-passtickets.md]: Thanks, updated

**Review by Shobhajayanna — APPROVED** [review]

**Comment by pablocarle** [comment]
> A couple of discussion points:
> 
>     * should we add a note in the doc that states how we used to require replay protection be disabled?
> 
>     * for folks that previously had added `APPLDATA('NO REPLAY PROTECTION')` - should they revert this change?
> 
> 
> For the first bullet, I think the answer is "no", but I'm curious for other's input. For the second, I'm not sure.

I think the second one makes sense, the only issue I see is that we just verified that it's not needed but there were no code changes recently. So, should we state that it's safe to enable the replay protection from 2.17 (latest release)?

**Pull request merged**